src/main/java/openblocks/NEIOpenBlocksConfig.java

```diff
@@ -34,4 +42,25 @@ public String getVersion() {
 		return "0.0";
 	}
 
+	private void API$hideItem(final ItemStack stack) {
```

Comment:
To split the class name from the method name.
The name is supposed to remember the call to the actual `API.hideItem()` method. Since using a dot is forbidden, I decided to go with a `$`, because it is rarely used and (usually) makes the separation visible.